scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh

```diff
@@ -17,24 +17,11 @@
 #*******************************************************************************
 
 # -------------------------------------------------------------------------
-# "IBM Storage Enabler for Containers" cli tool.
-# The tool is an helper to do the following actions : start, stop, status, collect_logs, sanity.
-# The script assume that you already installed ubiquity via the ubiquity_install.sh script.
+# IBM Storage Enabler for Containers CLI tool
+# This is a utility for starting/stoping IBM Storage Enabler for Containers (Ubiquity), retrieving its status, collecting logs, running sanity test.
+# The script asumes that you already installed Ubiquity using the ubiquity_install.sh script.
```

Comment:
1. Should limit line length to 80 characters.
2. I suggest making a general statement and then giving the options:
   This utility enables various operations with IBM Storage Enabler...:
   Start/Stop the service
   Get Status
3. asumes => assumes